### PR TITLE
onefetch: update windows-targets for gnullvm support

### PR DESCRIPTION
**Pull request by jeremyd2019**
Fixes https://github.com/msys2/MINGW-packages/pull/18083#issuecomment-1676330759
See https://github.com/o2sh/onefetch/issues/1143

Successful CLANGARM64 CI run: https://github.com/msys2-arm/MINGW-packages/actions/runs/5858975316/job/15884039932

### CODE REVIEW

File: mingw-w64-onefetch/PKGBUILD [hunk 1]
```diff
@@ -1,8 +1,8 @@
 _realname=onefetch
 pkgbase=mingw-w64-$_realname
 pkgname=$MINGW_PACKAGE_PREFIX-$_realname
-pkgver=2.17.0
-pkgrel=1
+pkgver=2.18.1
+pkgrel=2
```
Comment (Biswa96) [inline mingw-w64-onefetch/PKGBUILD:5]:
```suggestion
pkgrel=1
```

Comment (jeremyd2019) [reply mingw-w64-onefetch/PKGBUILD:5]: I bumped pkgrel after the revert of the revert :grin:

Comment (Biswa96) [reply mingw-w64-onefetch/PKGBUILD:5]: The current onefetch version is still 2.17.0-1 and this is a version bump.

Comment (jeremyd2019) [reply mingw-w64-onefetch/PKGBUILD:5]: But I believe 2.18.1-1 did exist in autobuild and/or staging for a while (until it was reverted after just clangarm64 failed to build) so it seemed safer to bump it to -2.

**Pull request merged**